libs/wingcompiler/src/compile.ts

```diff
@@ -294,6 +294,23 @@ npm i @wingcloud/framework
   }
 }
 
+/**
+ * Check if in the current working directory there is a default values file
+ * only the first match is returned from the list of default values files 
+ * 
+ * @returns default values file from the current working directory
+ */
+function defaultValuesFile() {
+  const defaultConfigs = [ "wing.toml", "wing.yaml", "wing.yml", "wing.json"]
```

Comment:
I need to add some docs explaining that these are the default files

Reply:
Should also be configurable and then you can specify the default in --help